lib/createSchemaMapperForVisitor.ts

```diff
@@ -26,6 +27,17 @@ export const createMapper = <T extends DirectiveLocation>(
     return mutation;
   },
   [MapperKind.OBJECT_TYPE](type, schema): GraphQLObjectType {
+    Object.values(type.getFields()).forEach(field => {
+      field.args.forEach(arg => {
+        const [directive] = getDirective(schema, arg, directiveName) ?? [];
+        if (!directive) return;
+        // eslint-disable-next-line no-param-reassign
+        visitor.args = directive;
```

Comment:
not sure, can't this cause problems ?
- e.g.:
- 1:
  - store the original value `before` the forEach loop
  - after the forEach loop, restore the original value, so the other mappers are not affected by the modification in the `args`
- 2:
  - use a different `visitor` instance to visit the fields 
- 3:
  - use `visitArgumentsWithDirectiveInObjectFields` (same code as this block)